cvxpy/lin_ops/canon_backend.py

```diff
@@ -62,14 +63,18 @@ def combine(cls, tensors: list[TensorRepresentation]) -> TensorRepresentation:
         Concatenates the row, col, parameter_offset, and data fields of a list of
         TensorRepresentations.
         """
+        start = time.time()
         data, row, col, parameter_offset = np.array([]), np.array([]), np.array([]), np.array([])
         # Appending to numpy arrays vs. appending to lists and casting to array at the end was
         # faster for relevant dimensions in our testing.
-        for t in tensors:
+        for i, t in enumerate(tensors):
+            print("size of tensor no", i, ":", t.data.shape[0])
```

Comment:
this print shows us the size of each tensor that is being passed to ``combine``. I find it really weird that there are such shape differences while not causing any issues to the problem data or final solution.